packages/bundler-sdk-viem/test/helpers.ts

```diff
@@ -180,11 +204,11 @@ export const setupTestBundle = async <chain extends Chain = Chain>(
         ),
       ]);
 
-      for (const { balance, adapter } of balances)
+      for (const { balance, balanceBefore, adapter } of balances)
         expect(
           balance,
           `balance of "${adapter}" for token "${token}"`,
-        ).toBeLessThanOrEqual(50n);
+        ).toBeLessThanOrEqual(balanceBefore + 5n);
```

Comment:
1. The conditional skimming doesn't depend on  the initial balance no?
2. Aren't we skimming in any case?